Podfile

```diff
@@ -35,7 +35,7 @@ def aztec
     #pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => ''
     #pod 'WordPress-Editor-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :tag => ''
     #pod 'WordPress-Editor-iOS', :path => '../AztecEditor-iOS'
-    pod 'WordPress-Editor-iOS', '1.19.7'
+    pod 'WordPress-Editor-iOS', '1.19.8'
```

Comment:
I had to bump the version of `WordPress-Editor-iOS` because it was complaining when installing the pods. This is most likely related to the changes introduced and afterwards reverted in the following PRs:
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17844
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17989